docs/index.rst

```diff
@@ -0,0 +1,42 @@
+.. alchemiscale-fah documentation master file, created by
+   sphinx-quickstart on Wed Feb 26 17:04:07 2025.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+#######################################################################
+high-throughput alchemical free energy execution...on Folding\@Home
```

Comment:
```suggestion
High-throughput alchemical free energy execution...on Folding\@Home
```
[nit]

Reply:
Haha this is my stylistic tendency for top-level titles; we have the same on the `alchemiscale` docs.